version_map.json

```
@@ -0,0 +1,6 @@
+{
+    "v1.2.2-beta": "v0.1.0",
+    "v1.2.3-beta": "v0.1.0",
+    "v1.2.4-beta": "v0.3.0",
+    "v1.2.5-beta": "v0.3.0"
```

```suggestion
    "v1.2.4-beta": "v0.3.1",
    "v1.2.5-beta": "v0.3.1"
```
just need to update these versions